src/app/stilling/[id]/_components/EmployerDetails.jsx

```diff
@@ -2,161 +2,119 @@ import React from "react";
-
-    for (let i = 0; i < locationList.length; i += 1) {
-        if (locationList[i].postalCode) {
-            let address = locationList[i].address ? `${locationList[i].address}, ` : "";
-            address += `${locationList[i].postalCode} ${fixLocationName(locationList[i].city)}`;
-            employerLocation.push(address);
-        } else if (locationList[i].municipal) {
-            employerLocation.push(`${fixLocationName(locationList[i].municipal)}`);
-        } else if (locationList[i].country) {
-            employerLocation.push(`${fixLocationName(locationList[i].country)}`);
-        }
-    }
-
-    return employerLocation.join(", ");
-}
-
-export default function EmployerDetails({ stilling }) {
-    const { properties } = stilling;
-    const employer = getEmployer(stilling);
-    const employerLocation = getEmployerLocation(stilling.employer);
+export default function EmployerDetails({ employer }) {
     return (
         <section className="mt-16 mb-16">
             <Heading level="2" size="large" spacing>
                 Om arbeidsgiveren
             </Heading>
             <dl className="dl mb-4">
-                {employer && (
+                {employer.name && (
                     <>
                         <dt>
                             <Label as="p">Arbeidsgiver</Label>
                         </dt>
                         <dd>
-                            <BodyLong>{employer}</BodyLong>
+                            <BodyLong>{employer.name}</BodyLong>
                         </dd>
                     </>
                 )}
-                {employerLocation && (
+                {employer.location && (
                     <>
                         <dt>
                             <Label as="p">Adresse</Label>
                         </dt>
                         <dd>
-                            <BodyLong>{employerLocation}</BodyLong>
+                            <BodyLong>{employer.location}</BodyLong>
                         </dd>
                     </>
                 )}
-                {properties.employerhomepage && (
+                {employer.homepage && (
                     <>
                         <dt>
                             <Label as="p">Hjemmeside</Label>
                         </dt>
                         <dd>
                             <BodyLong>
-                                {isValidUrl(properties.employerhomepage) ? (
-                                    <AkselLink href={properties.employerhomepage}>
-                                        {properties.employerhomepage}
-                                    </AkselLink>
+                                {isValidUrl(employer.homepage) ? (
+                                    <AkselLink href={employer.homepage}>{employer.homepage}</AkselLink>
                                 ) : (
-                                    properties.employerhomepage
+                                    employer.homepage
```

Comment:
denne kan vi vel slette for de den uansett i disse tilfellene er undefined og ingen ting vil rendres

Reply:
fixed